Tella/Data/Database/TellaDataBase.swift

```diff
@@ -100,6 +119,7 @@ class TellaDataBase {
     }
     
     func addServer(server : Server) throws -> Int {
+        dump(server)
```

Comment:
@dhekra-rouatbi I have added a new method using `overloading` so that we can even log the object types rather than only error and text. 

P.S changes are in uwazi/clean_up

Reply:
@rob-horizontal-iOS but I don't think that print the same format as dump, what do you think ?